README.md

```diff
@@ -119,7 +119,8 @@ After editing `/opt/harvest/harvest.yml`, manage Harvest with `systemctl start|s
 
 ## Docker
 
-Work in progress. Coming soon
+* Refer [One Poller One Container](docker/onePollerPerContainer/README.md)
+* Refer [All Poller In One Container](docker/allPollerInOneContainer/README.md)
```

Comment:
Let's only recommend one poller per container in the readme

Reply:
done.